cylc/rose/stem.py

```diff
@@ -17,11 +17,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""Cylc 8 implementation of 'rose stem'
-
-Provide a wrapper around "cylc install" for rose-stem suites.
-
-To run a rose-stem suite use "cylc play".
```

Comment:
This line would be useful to keep (old `rose-stem` users will get confused when nothing happens).